README.md

```diff
@@ -55,11 +34,14 @@ Example
 `crontab -e`
 ```
 SENTRY_DSN=https://<your_key>:<your_secret>@app.getsentry.com/<your_project_id>
-@reboot cron-sentry ./my-process
+cron-sentry my-process --arg arg2
```

Comment:
this line is missing the cron schedule time interval (originally it was on `@reboot`, but it could be something like `* * * * *`).